src/components/dropdown/styles.scss

```
@@ -26,11 +29,35 @@
       align-items: center;
       justify-content: center;
       width: 100%;
-      height: 24px;
+      height: 44px;
+      cursor: pointer;
+      background-color: $color-white;
+      border: {
+        left: 1px solid $color-gray-1;
+        right: 1px solid $color-gray-1;
+        top: 1px solid $color-gray-1;
+      }
+
+      &.disabled {
+        color: #ccc;
+        cursor: not-allowed;
+      }
 
       &:hover {
         background-color: darken($color-background, 10%);
       }
     }
+
+
+  }
+}
+
+.dd-list {
```

Instead of using `& > *:last-child > *:last-child ` I'd suggest to set border on the `ul` element (list), and `border-bottom` to the `li` elements (list items)

```suggestion
.dd-list {
  border: 1px solid $color-gray-1;
  & > *:not(:last-child) {
    border-bottom: 1px solid $color-gray-1;
  }
}
```

---

👍